### PR TITLE
AdminPanel: Fix menu for host users

### PR DESCRIPTION
**Pull request by Betree**
Based on the feedback from https://opencollective.slack.com/archives/C0RMV6F8C/p1635844010014900

This PR adds the general menu entries for host users.

image

### CODE REVIEW

File: components/admin-panel/Menu.js [hunk 1]
```diff
@@ -74,7 +75,7 @@ const Menu = ({ collective }) => {
           </SubMenu>
           <SubMenu
             label={<FormattedMessage id="AdminPanel.FiscalHostSettings" defaultMessage="Fiscal Host Settings" />}
-            if={isHost || (isType(collective, USER) && isHost)}
+            if={isHost || (isIndividual && isHost)}
```
Comment (znarf) [inline components/admin-panel/Menu.js:78]: If I read the code, seems `isHost` should be enough?

File: components/admin-panel/Menu.js [hunk 2]
```diff
@@ -111,8 +112,8 @@ const Menu = ({ collective }) => {
           <OrganizationSettingsMenuLinks collective={collective} />
         </MenuGroup>
 
-        {/** General settings for other (non-host) profiles */}
-        <MenuGroup if={!isHost && !isType(collective, ORGANIZATION)}>
+        {/** General settings for other profiles */}
+        <MenuGroup if={!isType(collective, ORGANIZATION) && (!isHost || isIndividual)}>
```
Comment (znarf) [inline components/admin-panel/Menu.js:116]: It's a bit difficult to parse, can we explain this condition in plain english in a comment?

File: components/admin-panel/Menu.js (hunk 1 above)
Comment (Betree) [reply components/admin-panel/Menu.js:78]: Definitely. I did a simple search & replace on `isType(collective, USER)`, did not notice that the condition was redundant

File: components/admin-panel/Menu.js (hunk 2 above)
Comment (Betree) [reply components/admin-panel/Menu.js]: I simplified the conditions and updated the comments

Comment (znarf) [reply components/admin-panel/Menu.js]: Nice

**Review by znarf — APPROVED** [review]
Looks good, I haven't tested.
